source/nodejs/README.md

```diff
@@ -19,9 +19,23 @@ We use lerna to manage package linking and building.
 2. `npm install`
 3. `npx lerna bootstrap`
 
+## Initialize Telemetry
+
+We use Microsoft 1DS as our telemetry provider.
```

Comment:
>e use Microsoft 1DS as our telemetry provider [](start = 1, length = 45)

More like "A provider that allows for sending telemetry data to the internal Microsoft service endpoints directly"